python/bump-and-tag/action.yml

```diff
@@ -0,0 +1,50 @@
+name: Python Bump and Tag
+description: Perform bump and tag operations for Python Libraries
+inputs:
+  version:
+    description: "The published version"
+    required: true
+  post_version:
```

Comment:
`post_version` is a little ambiguous: I'd prefer to rename this something more explicit like `following_version`.

Reply:
Done